source/Octopus.Server.Client/Model/ModifyRunbookProcessCommand.cs

```diff
@@ -0,0 +1,7 @@
+namespace Octopus.Client.Model
+{
+    public class ModifyRunbookProcessCommand : RunbookProcessResource, ICommitCommand
```

Comment:
This type is only used for Git runbook processes, is that right? I wonder if its worth putting `Git` in the name e.g. `ModifyGitRunbookProcessCommand`. Same for the other one above.

Reply:
The naming is consistent with deployment process, deployment settings, etc, which is why I went with this specific name. I see an argument either way.